### ISSUE TITLE
Filter on indicator for sex workers does not work

### ISSUE THREAD

**Issue by nsoko** [BlueCodeSystems/nacmis#598]
![image](https://user-images.githubusercontent.com/38098694/65223057-a5198580-dac0-11e9-86c0-8df5938ac383.png)
Filter on indicator for sex workers does not work

**Comment by nsoko** [comment]
Labels  for the Quaters  selected on graph for sex workers and mobile population to be included.

**Comment by Mellorison** [comment]
Sorry, Which Filter exactly?

**Comment by nsoko** [comment]
yes in the Key population dashboard its not filters because the numbers are not changing  when u select diffrent quaters

**Comment by nsoko** [comment]
The Filters for quaters
